packages/block-editor/src/components/block-actions/index.js

```diff
@@ -165,6 +167,14 @@ export default compose( [
 					innerBlocks
 				);
 			},
+			onToggleBlockMode() {
+				const block = blocks[ 0 ];
+				const isCodeEditingEnabled = select( 'core/editor' ).getEditorSettings().codeEditingEnabled;
+
+				if ( hasBlockSupport( block.name, 'html', true ) && isCodeEditingEnabled && ! isLocked && ! isReusableBlock( block ) ) {
+					toggleBlockMode( clientIds );
```

Comment:
@talldan should I use clientIds[0] to be consistent `const block = blocks[ 0 ];`?